README.md

```diff
@@ -46,8 +46,73 @@ be assigned. |
 
 `user_visits_form_smoke_spec.rb` will run an end to end acceptance test, creating a real ticket in Zendesk when this test submits the config form.
 
-The test is run within Docker. This allows us to create an environment that includes the dependencies required by Selenium to run on our Jenkins setup. The Jenkins job that runs the smoke test is triggered every 5 minutes and after this application is deployed.
+The test is run within Docker. This allows us to create an environment that includes the dependencies required by Selenium to run on our Jenkins setup. The [Jenkins job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-smoke-test/) that runs the smoke test is triggered every 5 minutes and after this application is deployed.
 
 The smoke test has been filtered out of the main test run (`bundle exec rspec spec`) via a `smoke` filter.
 
 To run the smoke test, use: `./smoke-test.sh`.
+
+
+## Deploying
+
+The app is deployed to the PaaS and is called `verify-environment-access`. The production space is called `docs`. There are two other spaces, `onboarding-forms` and `sandbox`, but the app is not currently deployed to those.
```

Comment:
it is on onboarding-forms as `verify-environment-access-staging`. There are any other spaces under Verify, I just only gave you access to those (stubs is deployed on PaaS, for example)